cypress/e2e/edit-profile.cy.js

```diff
@@ -0,0 +1,69 @@
+describe('Edits profile details', () => {
+  it('should check that profile details are successfully edited', () => {
+    // Intercept the PUT request and login API call
+    cy.intercept('PUT', 'https://cors.noroff.dev/https://agency-api.noroff.dev/users/*').as(
```

Comment:
Good job in cleaning it up! 🚀 
Now that we got the new assertions, it looks like we no longer need the 'intercept''s anymore